source/jsonnet/northern-ireland/individual/blocks/identity-and-health/religion.jsonnet

```diff
@@ -94,6 +94,12 @@ local question(title) = {
         when: [rules.under3],
       },
     },
+    {
+      goto: {
+        block: 'health',
+        when: [rules.lastBirthdayAgeUnder(3)],
+      },
+    },
```

Comment:
also needs a when rule to check that none wasn't selected

Reply:
It doesnt need to, because thee routing about says go to `no-religion` if `religion-answer` is not set, which implies, the exclusive answer is either chosen, or they haven't answered.

Reply:
I am confused, the card states
"religion - if under 3 and answer is not 'none' or unanswered then route to health. If 3+ and answer is not 'none' or unanswered then route to language."
When I was 17 days old (from today, but still under 3 come census) and didn't answer I went to no-religion which according to the above is wrong. Is the card wrong?